### ISSUE TITLE
Config value `min_max_default_coins:` is used "incorrectly"

### ISSUE THREAD

**Issue by Czekoloczek** [Mikihero/BetterCoinflips#14]
If you set the value to
```
min_max_default_coins:
- 1
- 1
```
the coin will have 2 throws

but if you set the value to
```
min_max_default_coins:
- 0
- 0
```
the coin will have 1 throw

Video of the bug:
https://github.com/Mikihero/BetterCoinflips/assets/76474182/4a159ffc-1cdd-4b89-abcd-17d8223b3449

**Comment by Mikihero** [comment]
Both of these are a result of me failing to use a certain move called "brain" while coding that part. I've spotted both the issues  and will fix it once I'm home.

**Comment by Mikihero** [comment]
fixed in 3.0.3
